packages/libs/react-ui/src/components/Card/Card.stories.tsx

```diff
@@ -5,22 +5,48 @@ import React from 'react';
 
 const meta: Meta<ICardProps> = {
   title: 'Layout/Card',
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'A component used for grouping se of items in a card.\n' +
+          '\nThis component allows for passing the <i>stack</i>, <i>fullWidth</i>,  and <i>disabled</i> properties.',
+      },
+    },
+  },
   component: Card,
   argTypes: {
     stack: {
       control: {
         type: 'boolean',
       },
+      description:
+        'If true, the component stacks the child elements vertically.',
```

Comment:
Actually the stack property allows users to vertically stack multiple card together and applies styles that combine them into a single card with separators
<img width="692" alt="image" src="https://github.com/kadena-community/kadena.js/assets/9022549/34163212-308e-4b39-b6be-df99d3e498a5">